src/lib/model.ts

```diff
@@ -271,13 +305,18 @@ export class ModuleWithCourse extends Module {
 export class Event {
 	static create(data: any) {
 		// TODO: Matt - this is a temp work around to circumvent new event definition not matching UI
-		let date: any = ''
+		let startDate: any = ''
+		let endDate: any = ''
+
 		if (data.dateRanges[0]) {
-			date = new Date(
+			startDate = new Date(
 				data.dateRanges[0].date + 'T' + data.dateRanges[0].startTime
 			)
+			endDate = new Date(
+				data.dateRanges[0].date + 'T' + data.dateRanges[0].endTime
+			)
 		} else {
-			date = data.date
+			// date = data.date
```

Comment:
remove else if not needed.  branch which does nothing as you have commented the only line